Documents/Documents/Code/Controllers/Documents/ASCDocumentsViewController.swift

```diff
@@ -1352,7 +1352,7 @@ class ASCDocumentsViewController: ASCBaseTableViewController, UIGestureRecognize
 
         addBarButton?.isEnabled = !hasError && provider?.allowAdd(toFolder: folder) ?? false
         if let folder = folder,
-           !folder.isRoom
+           !folder.isRoom || folder.rootFolderType == .onlyofficeRoomArchived
```

Comment:
it looks like we can return false in provider?.allowAdd instead checking an archive here

Reply:
Now used provider?.allowAdd for checking